pages/common/apropos.md

```diff
@@ -1,15 +1,16 @@
 # apropos
 
-> Search in manpages, for example to find a new command.
+> Search the manual pages for names and descriptions.
+> More information: <https://man.archlinux.org/man/apropos.1>.
 
-- Search for keyword:
+- Search for a keyword using a regular expression:
 
 `apropos {{regular_expression}}`
 
-- Search without restricting output to terminal width:
+- Search without restricting the output to the terminal width:
 
 `apropos -l {{regular_expression}}`
 
-- Search for pages that only contain all of the expressions given (AND search):
+- Search for pages only that contain all of the expressions given (AND search):
```

Comment:
Not sure about this grammar @bl-ue

Reply:
Yeah, the original sounds better (`that only`).  How about this:
```suggestion
- Search for pages that contain all of the expressions given:
```